common/actions.js

```diff
@@ -1,6 +1,8 @@
 import LocalStorage from '../utils/LocalStorage'
+import { buildIntentsDataFromCyElements } from '../cy_canvas/helper'
 
 let intentId = parseInt(LocalStorage.getElements("elementId"));
+const devKey = LocalStorage.getDevKey();
```

Comment:
I noticed that before but I thought it was minor. However, it's not too hard to fix it and that's what I'm gonna do.